include/compress.h

```diff
@@ -0,0 +1,25 @@
+#ifndef __CODEC_H__
```

Comment:
This should be __COMPRESS_H__ not __CODEC_H__

Reply:
😱